src/main/java/me/totalfreedom/totalfreedommod/fun/Trailer.java

```diff
@@ -1,10 +1,9 @@
 package me.totalfreedom.totalfreedommod.fun;
 
-import java.util.HashSet;
-import java.util.Objects;
-import java.util.Set;
-import java.util.SplittableRandom;
+import java.util.*;
```

Comment:
Can we un-do the wildcard import. I seem to recall it being bad practice.

Reply:
Intellij thought it would be charitable to do this automatically. Let me go fix that.

Reply:
Fixed